frontend/src/features/room/lobby/components/TabBar/TabBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { useRef, useEffect, useCallback } from 'react';
+import * as S from './TabBar.styled';
+
+type Props = {
+  tabs: string[];
+  activeTab: number;
+  onTabChange: (index: number) => void;
+};
+
+const TabBar = ({ tabs, activeTab, onTabChange }: Props) => {
+  const tabRefs = useRef<(HTMLButtonElement | null)[]>([]);
+  const indicatorRef = useRef<HTMLDivElement>(null);
```

Comment:
여기도 tabRefs랑 일관성있게 타입을 가져가는건 어떨까요?

```suggestion
  const indicatorRef = useRef<HTMLDivElement | null>(null);
```